titler-cashman/styles/layout.css

```diff
@@ -0,0 +1,18 @@
+.heading-group-h1{
+  display:inline-block;
+  float: left;
+ 
+}
+
+.icon-menu{
+  float:right;
+  padding:10px;
+}
+
```

Comment:
Things like this h1 and main I'd probably put in base since they effect every h1 and main element on the page.